### PR TITLE
docs: added +lsp flag to terraform module

### PR DESCRIPTION
**Pull request by ponchoalv**
Replace this with a summary of what you've changed and why, followed by a list of issues it affects, if any.

Fix: #0000
Ref: #0000
Close: #0000

-----
- [x] I searched the issue tracker and this hasn't been PRed before.
- [x] My changes are not on [the do-not-PR list](https://doomemacs.org/d/do-not-pr) for this project.
- [x] My commits conform to [the git conventions](https://doomemacs.org/d/git-conventions).
- [x] My changes are visual; I've included before and after screenshots.
- [ ] I am blindly checking these off.
- [x] Any relevant issues or PRs have been linked to.
- [x] This a draft PR; I need more time to finish it.

### CODE REVIEW

File: modules/tools/terraform/README.org [hunk 1]
```diff
@@ -6,13 +6,15 @@
 * Description :unfold:
 This module adds support for working with [[https://www.terraform.io][Terraform]] files within Emacs. This
 includes syntax highlighting, intelligent code completion, and the ability to
-run Terraform commands directly from Emacs.
+run Tearraform commands directly from Emacs.
```
Comment (hlissner) [inline modules/tools/terraform/README.org:9]: Please revert this typo.

**Review by hlissner — CHANGES_REQUESTED** [review]
To conform to [our git conventions](https://doomemacs.org/d/git-conventions), please reword the commit to:

```
docs(terraform): mention +lsp flag
```

Side-note:

(¬‿¬) image

File: modules/tools/terraform/README.org (hunk 1 above)
Comment (ponchoalv) [reply modules/tools/terraform/README.org]: done, thank you!

**Comment by ponchoalv** [comment]
> Side-note:
> 
> (¬‿¬) image

:trollface:

**Comment by hlissner** [comment]
Excellent! Thanks for your help!

**Pull request merged**